torch/csrc/jit/interned_strings.h

```diff
@@ -102,6 +111,11 @@ _(onnx, Transpose) \
 #define FORALL_ATTR_EXTRA_SYMBOLS(_) \
 _(attr, Subgraph) \
 _(attr, axis) \
+_(attr, axes) \
+_(attr, direction) \
+_(attr, input_as_shape) \
+_(attr, starts) \
+_(attr, ends) \
```

Comment:
I think those are sorted at the moment, and it would be better to keep them like this (to avoid duplicates later).

Reply:
++ will sort